conanfile.py

```diff
@@ -12,17 +12,27 @@ class CppUTest(ConanFile):
     source_dir = "{name}-{version}".format(name=name, version=version)
     options = {
         "shared": [True, False],
-        "include_pdbs": [True, False],
         "fPIC": [True, False],
-        "tests": [True, False],
-        "extensions": [True, False]
+        "use_std_c_lib": [True, False],
+        "use_std_cpp_lib": [True, False],
+        "use_cpp11": [True, False],
```

Comment:
I advise to use `settings.cppstd` to choose the appropriate language standard ([\[Conan.io\] How to manage C++ standard](https://docs.conan.io/en/latest/howtos/manage_cpp_standard.html))

Reply:
I like the idea, but (1) that setting is in "experimental" stage, (2) the user has to specify it manually, and (3) it maps so poorly to the way that the CppUTest CMake logic works that it would require many lines of convoluted logic to reduce it to the single CppUTest configuration.

All that said, I'm going to extricate myself from the rabbit hole into which I got myself down.